test/index.test.js

```diff
@@ -1008,7 +1007,7 @@ describe('General error handling', function () {
     expect(response.json()).toEqual({
       statusCode: 401,
       error: 'Unauthorized',
-      message: 'Authorization header should be in format: Bearer [token].'
+      message: 'Missing Authorization HTTP header.'
```

Comment:
This feels like oauth2-mock-server matured its error messages but I need to dig into to make sure.  Hence the draft.

Reply:
This test change is the result of upgrading @fastify/jwt.

REL: https://github.com/fastify/fastify-jwt/pull/325